src-vue/src/utils/NorthstarMod.d.ts

```diff
@@ -1,6 +1,6 @@
 // Matches Rust struct (in lib.rs).
 export interface NorthstarMod {
-    name: String,
-    thunderstore_mod_string?: String,
+    name: string,
+    thunderstore_mod_string?: string,
```

Comment:
`String` is the JavaScript class that allows you to create strings (e.g. `const s = new String("hello there")`), which nobody uses since you don't need it (e.g. `const s = "hello there"`).

On the other hand, `string` is a TypeScript type.

Reply:
ugh, I swear I'm gonna rewrite frontend in pure webassembly at some point just so I don't have to deal with JS/TS stuff >:)

Reply:
You don't have a choice for now, unfortunately 😄